tests/app/main/test_services.py

```diff
@@ -1451,8 +1422,8 @@ def test_unanswered_questions_count(self, count_unanswered, data_api_client):
         count_unanswered.return_value = 1, 2
         res = self.client.get('/suppliers/frameworks/g-cloud-7/submissions/scs/1')
 
-        assert_true(u'3 unanswered questions' in res.get_data(as_text=True),
-                    "'3 unanswered questions' not found in html")
+        assert u'3 unanswered questions' in res.get_data(as_text=True), \
+            "'3 unanswered questions' not found in html"
```

Comment:
Yeah maybe the original author thought it was a particularly hard to track down eroor message by default